tests/PassphraseGeneratorTest.php

```
@@ -452,6 +453,13 @@ public function test_target_entropy_bits_overrides_num_words_param(): void
         $this->assertCount($expectedNumWords, explode('-', $result));
     }
 
+    public function test_target_entropy_bits_with_single_word_list_throws_word_list_exception(): void
+    {
+        $this->expectExceptionObject(WordListException::insufficientEntropy());
+
+        new PassphraseGenerator(WordList::fromArray(['only']));
+    }
```

PHPMD (2.15.0)

[error] The method test_target_entropy_bits_with_single_word_list_throws_word_list_exception is not named in camelCase. (undefined)

(CamelCaseMethodName)